mcd/auctions.md

```diff
@@ -76,18 +76,21 @@ Bids are created on `kick` and updated with every subsequent bid event.
 | Flipper.deal   | DSNote    | Flipper.bids[e.id]  | BidEvent        |
 | Flipper.yank   | DSNote    | Flipper.bids[e.id]  | BidEvent        |
 | Flopper.kick   | Kick      | Flopper.bids[e.id]  | BidEvent        |
+| Flopper.tick   | DSNote    | Flopper.bids[e.id]  | BidEvent        |
 | Flopper.tend   | DSNote    | Flopper.bids[e.id]  | BidEvent        |
 | Flopper.deal   | DSNote    | Flopper.bids[e.id]  | BidEvent        |
 | Flopper.yank   | DSNote    | Flopper.bids[e.id]  | BidEvent        |
 | Flapper.kick   | Kick      | Flapper.bids[e.id]  | BidEvent        |
+| Flapper.tick   | DSNote    | Flapper.bids[e.id]  | BidEvent        |
```

Comment:
Sorry to leave this hanging for awhile, but the API should currently be exposing `tick` events for both flap and flop 👍